<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="1" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-14T18:38:48 IST" name="Default suite" finished-at="2021-08-14T18:43:57 IST" duration-ms="308439">
    <groups>
    </groups>
    <test started-at="2021-08-14T18:38:48 IST" name="Default test" finished-at="2021-08-14T18:43:57 IST" duration-ms="308439">
      <class name="BeginTest.SignUpTest">
        <test-method is-config="true" signature="openWebPage()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:38:50 IST" name="openWebPage" finished-at="2021-08-14T18:39:47 IST" duration-ms="56786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openWebPage -->
        <test-method signature="SignUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:39:47 IST" name="SignUp" data-provider="SignUpData" finished-at="2021-08-14T18:39:53 IST" duration-ms="5680" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Anurag]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Yadav]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9125185579]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[anuragyadav@gmai.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Customer]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TestPass]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at BeginTest.SignUpTest.SignUp(SignUpTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:39:53 IST" name="teardown" finished-at="2021-08-14T18:39:56 IST" duration-ms="3408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="openWebPage()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:39:56 IST" name="openWebPage" finished-at="2021-08-14T18:40:58 IST" duration-ms="61999" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openWebPage -->
        <test-method signature="SignUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:40:58 IST" name="SignUp" data-provider="SignUpData" finished-at="2021-08-14T18:41:04 IST" duration-ms="5654" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Rashmi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Yadav]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9559866689]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[rashmi.pooja2009@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Customer]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TestPass]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at BeginTest.SignUpTest.SignUp(SignUpTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:41:04 IST" name="teardown" finished-at="2021-08-14T18:41:07 IST" duration-ms="2716" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="openWebPage()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:41:07 IST" name="openWebPage" finished-at="2021-08-14T18:41:58 IST" duration-ms="51492" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openWebPage -->
        <test-method signature="SignUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:41:58 IST" name="SignUp" data-provider="SignUpData" finished-at="2021-08-14T18:42:04 IST" duration-ms="5737" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Lajja]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Yadav]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9335464773]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[lajja.yadav@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Customer]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TestPass]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at BeginTest.SignUpTest.SignUp(SignUpTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:42:04 IST" name="teardown" finished-at="2021-08-14T18:42:06 IST" duration-ms="2633" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="openWebPage()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:42:06 IST" name="openWebPage" finished-at="2021-08-14T18:42:51 IST" duration-ms="44573" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openWebPage -->
        <test-method signature="SignUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:42:51 IST" name="SignUp" data-provider="SignUpData" finished-at="2021-08-14T18:42:57 IST" duration-ms="5661" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Virat]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Yadav]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9503766689]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[ramlal.yadav@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Customer]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TestPass]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [PHPTRAVELS - PHPTRAVELS] but found [Signup - PHPTRAVELS]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at BeginTest.SignUpTest.SignUp(SignUpTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:42:57 IST" name="teardown" finished-at="2021-08-14T18:43:00 IST" duration-ms="2907" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="openWebPage()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:43:00 IST" name="openWebPage" finished-at="2021-08-14T18:43:48 IST" duration-ms="48185" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openWebPage -->
        <test-method signature="SignUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:43:48 IST" name="SignUp" data-provider="SignUpData" finished-at="2021-08-14T18:43:54 IST" duration-ms="6233" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Abhishek]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Yadav]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[8400029027]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[abhishek.yadav@gmail.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Customer]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TestPass]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:BeginTest.SignUpTest@55b53d44]" started-at="2021-08-14T18:43:54 IST" name="teardown" finished-at="2021-08-14T18:43:57 IST" duration-ms="2642" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- BeginTest.SignUpTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
